src/components/shoot-controls.js

```diff
@@ -58,8 +58,6 @@ AFRAME.registerComponent('shoot-controls', {
   update: function () {
     var data = this.data;
     var el = this.el;
-    // handId: 0 - right, 1 - left
-    var controller = data.hand === 'right' ? 0 : 1;
-    el.setAttribute('tracked-controls', 'controller', controller);
+    el.setAttribute('auto-detect-controllers', {hand:data.hand, trackedcontrols:true});
```

Comment:
(and I think having some intermediary may scale better, e.g. this may allow `gearvr-controls` and daydream to work... will have to try that later)